sympy/combinatorics/tests/test_homomorphisms.py

```diff
@@ -57,6 +57,13 @@ def test_homomorphism():
     assert T(a*b) == p
 
 def test_isomorphisms():
+
+    # Infinite ordered groups where relators arrangement is different.
```

Comment:
"with differently ordered relators" is clearer. Also, saying `infinite group` is sufficient.